elucideye/drishti
# add private drishti_gltest lib for cross platform opengl context

NOTE: iOS and Android still need a non QT solution

* Add drishti_gltest for portable windowless GLContext
  + GLFWContext (NOT (IOS OR ANDROID))
  + QGLContext (NOT (IOS OR ANDROID)), but should most likely work
* global query replace in filenames, C++, and CMakeLists.txt: ’s/qtplus/gltest/g’
* disable qt + mobile (IOS OR ANDROID) GPU gtests for drishti_{hci,acf,drishti}
* control clang formatting w/ on/off switches
* disable logging during gtests
  + add —silent option to turn loggin goff
  + add option to name logger for FaceTracker
  + remove unused threshold shader param for Nms2
